### DISCUSSION TITLE
Total Experience Gain From Campaign For New Players On Server

### DISCUSSION THREAD

**Discussion by PitucoSP** [FakeFishGames/Barotrauma#8852] [Ideas and suggestions]
An option for the server hosts enable for new players that come to the server, they could receive a specify % of the total exp through the hole campaign. Giving a chance for new players almost receive a fair gameplay instead of just watching others highly talent players enable to do almost all the work on subs while the others are almost incapable to do anything, plus giving a rest from captains begin asked many times for medals too... hehe

Reason behind a lot of servers that sometimes are really close to late game or games that players just join to have fun, suddenly feel useless for joining late games or else just sad that sometimes servers disappears and lose all the progress from hard work and trying to get their talent points.

The point of this feature is to give a little help for those new players to have a chance to a least have 2 or 3 talents point depending how much of the total exp the hole server gained or by the amount of missions completed. This is only a option for server host to enable if they want at all.

**Comment by mLuby** [comment]
That's reasonable. On the other hand, there exist other ways to accomplish this goal:

Using talents, the new player can join as a [Grayshirt Assistant](https://barotraumagame.com/wiki/Talents#Assistant_Talents) with [Playing Catchup](https://barotraumagame.com/wiki/Playing_Catchup) and [Graduation Ceremony](https://barotraumagame.com/wiki/Graduation_Ceremony) talents. Captains with the [Trusted Captain](https://barotraumagame.com/wiki/Trusted_Captain)/[Trucker](https://barotraumagame.com/wiki/Trucker)/[Esteemed Captain](https://barotraumagame.com/wiki/Esteemed_Captain) talents can give the new character [Coalition Commendations](https://barotraumagame.com/wiki/Coalition_Commendation) and [Coalition Medals](https://barotraumagame.com/wiki/Coalition_Medal) to instantly boost the new character's experience.

Or if a server admin is willing to cheat, they can use the console command [`giveexperience [amount] [character name]`](https://barotraumagame.com/wiki/Console_Commands#:~:text=giveexperience) to bring them up to the crew's level. For a less blatantly cheating option, the server admin could `spawnitem` Coalition Medals into the Captain's inventory for pinning onto the new character's chest.
